src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
The link to the stylesheet is not correct. HTML doesn't understand SCSS files. You should compile your SCSS to CSS and then link the resulting CSS file in your HTML.